core/testcontainers/core/config.py

```diff
@@ -73,8 +76,19 @@ def docker_auth_config(self, value: str):
         self._docker_auth_config = value
 
     def tc_properties_get_tc_host(self) -> Union[str, None]:
+        if "tc_properties_get_tc_host" in _WARNINGS:
+            warning(_WARNINGS.pop("tc_properties_get_tc_host"))
         return self.tc_properties.get("tc.host")
 
+    @property
+    def tc_properties_tc_host(self) -> Union[str, None]:
+        return self.tc_properties.get("tc.host")
+
+    @property
+    def tc_properties_testcontainers_reuse_enable(self) -> bool:
+        enabled = self.tc_properties.get("testcontainers.reuse.enable")
+        return enabled == "true"
```

Comment:
`testcontainers.reuse.enable` (like the Java implementation) or `tc.reuse.enable` (like `tc.host`)?

`enable` (like the Java implementation) or `enabled` (like RYUK_DISABLED)?

Reply:
Let's keep the same key as Java implementation `testcontainers.reuse.enable`.

Reply:
Yes, this is important. This key is used across most language implementations and the way tc-java does is considered the reference implemetation.

Reply:
Okay, then let's keep it as implemented.